app/lib/actions/katello/repository/update.rb

```diff
@@ -50,9 +50,15 @@ def plan(root, repo_params)
 
         def run
           repository = ::Katello::Repository.find(input[:repository_id])
+          output[:repository_id] = input[:repository_id]
           ForemanTasks.async_task(Katello::Repository::MetadataGenerate, repository)
         end
 
+        def finalize
+          repository = ::Katello::Repository.find(output[:repository_id])
+          ForemanTasks.async_task(::Actions::Katello::Repository::UpdateCVRepoCertGuard, repository, SmartProxy.pulp_master)
```

Comment:
is it possible to only do this if we detect changes to the 'unprotected' attribute?